### ISSUE TITLE
A question from the paper

### ISSUE THREAD

**Issue by princewen** [racp-kdd/racp#1]
In paper《Modeling Users’ Contextualized Page-wise Feedback for Click-Through Rate Prediction in E-commerce Search》，in formula(4)，Should pt be p(t+1)，since pt is calculated by Qt？

**Comment by yilifzf** [comment]
Yes, you are right. Really thanks for pointing out the typo and the nice introduction of this work in your blog. We may release the revised version in the arXiv in the future.
